cmd/plotFunctions/plotFunctions.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/numbers"
+	"github.com/vertgenlab/gonomics/popgen"
+	"github.com/vertgenlab/gonomics/common"
+	"log"
+	"fmt"
+	"flag"
+	"math"
+)
+
+func plotFunctions(function string, left float64, right float64, bins int, outFile string, alpha float64) {
+	if function == "AfsStationarity" {
+		numbers.Plot(popgen.AFSStationarityClosure(alpha), left, right, bins, outFile)
+	} else {//here you can add more else ifs to add additional functions for plotting
+		fmt.Printf("Unrecognized function: %s.\n", function)
+	}
+}
+
+func usage() {
+	fmt.Print(
+		"plotFunctions-returns a tab separated list of function evaluations for plotting functions.\n" +
+		"To specify the function, use a function keyword. Currently only 'AfsStationarity' is supported.\n" + 
+			"Usage:\n" +
+			" plotFunctions functionKeyWord leftBound rightBound numberOfBins outFile\n" +
```

Comment:
Is "bins" what we want?  From a quick scan of the code, maybe "stepSize" or "pointsToPlot" would be better?  I think of "bin" as summarizing data over an interval (the width of the bin"), but I think this program is giving me the function value at certain points e.g. f(leftBound), f(leftBound+step), etc